pyqt-apps/siriushla/bl_ap_imgproc/main.py

```diff
@@ -305,6 +305,9 @@ def update_bl_open_led(self):
             self.pydm_led.value_changed(status_bl)
             if old_val != status_bl:
                 self.end_processing_cmd()
+        if self.pydm_lbl != None:
```

Comment:
This if doesn't seem to be needed

Reply:
Oh, I see. It is needed because timer is created before setting up the widgets...